src/test/java/seedu/duke/database/DataBaseTest.java

```diff
@@ -0,0 +1,92 @@
+            System.out.println("Input: Chicken , Ayam Penyet ## OutPut: " + database.searchFoodByNameByStore("Chicken", "Ayam Penyet"));
+            System.out.println("Input: lobster , Michelin ## OutPut: " + database.searchFoodByNameByStore("lobster", "Michelin"));
+            System.out.println("Input: fish , Halal Mini Wok ## OutPut: " + database.searchFoodByNameByStore("fish", "Halal Mini Wok"));
+        }
+        catch (NoSuchElementException e){
+            System.out.println("No such food found! " + e);
+        }
+
+        // ---- search all food by store -----
+        try {
+            System.out.println("------- testing the search ALL food by store and by name function --------");
+            System.out.println("------- Input: Halal Mini Wok -------- ");
+            database.searchAllFoodByStore("Halal Mini Wok").forEach(System.out::println);
+            System.out.println("------- Input: Ayam Penyet -------- ");
+            database.searchAllFoodByStore("Ayam Penyet").forEach(System.out::println);
+            System.out.println("------- Input: Michelin -------- ");
+            database.searchAllFoodByStore("Michelin").forEach(System.out::println);
+            System.out.println("------- Input: Gordan Ramsey's restaurant -------- ");
+            database.searchAllFoodByStore("Gordon Ramsey's restaurant").forEach(System.out::println);
+        }
+        catch (NoSuchElementException e) {
+            System.out.println("There is no such store! " + e);
+        }
+
+        // ---- search food by Name by canteen ------
+        try {
+            System.out.println("------- testing the search food by canteen and by name function --------");
+            System.out.println("Input: Fried , Science ## OutPut: " + database.searchFoodByNameByCanteen("Fried", "Science"));
+            System.out.println("Input: Chicken , Science ## OutPut: " + database.searchFoodByNameByCanteen("Chicken", "Science"));
+            System.out.println("Input: lobster , Science ## OutPut: " + database.searchFoodByNameByCanteen("lobster", "Science"));
+            System.out.println("Input: lobster , Raffles Hotel Suite ## OutPut: " + database.searchFoodByNameByCanteen("lobster", "Raffles Hotel Suite"));
+        }
+        catch (NoSuchElementException e) {
+            System.out.println("There is either no such canteen or no such food in that canteen!" + e);
+        }
+
+        // ---- search all food below calorie ------
+
+        System.out.println("------- testing the search food below calorie function --------");
+        System.out.println(" ---- Input : 400");
+        database.searchAllFoodBelowCalorie(400).forEach(System.out::println);
+        System.out.println(" ---- Input : 200");
+        database.searchAllFoodBelowCalorie(200).forEach(System.out::println);
+        System.out.println(" ---- Input : 3428");
+        database.searchAllFoodBelowCalorie(3428).forEach(System.out::println);
+
+    }
+}
```

Comment:
Perhaps you can consider splitting them up into different tests using assertEquals and assertThrows for tests that throw exceptions.